cmd/oras/manifest/push.go

```diff
@@ -75,19 +74,22 @@ Example - Push a manifest to repository 'locahost:5000/hello' and tag with 'tag1
 
 Example - Push a manifest to repository 'locahost:5000/hello' and tag with 'tag1', 'tag2', 'tag3' and concurrency level tuned:
   oras manifest push --concurrency 6 localhost:5000/hello:tag1,tag2,tag3 manifest.json
+
+Example - Push a manifest to an OCI layout folder 'layout-dir' and with tag 'latest':
```

Comment:
Should we use `tag with 'latest'` to keep consistent with other examples?

Reply:
I will update all tags to 'latest' to make it consistent.

Reply:
Sorry I guess I misunderstand your meaning, will rephrase to `tag with`